torch_xla/csrc/init_python_bindings.cpp

```diff
@@ -1537,6 +1537,57 @@ void InitXlaModuleBindings(py::module m) {
     }
     return std::string();
   });
+  m.def("_get_local_shards",
```

Comment:
Add a comment saying that the returned shards are padded, while the indices are not, excluding the pads.